### ISSUE TITLE
Shortform videos

### ISSUE THREAD

**Issue by ferdiworks** [rrirower/DP2SceneGraphWizard#8]
While I have successfully converted many channels to date with Alpha3 version for tvspecials I have not been able to successfully convert any shortform vdeo feeds running the software. But when I have edited the source code myself I got it to parse shortform videos on my own. But to run the software as is and use shortform videos feeds I have not had success!

**Comment by rrirower** [comment]
Can you explain what you mean by "I have not had success".  Please describe in more detail.

Additionally, if you can provide a feed URL to test with, I can debug deeper.

**Comment by Dalidadabro** [comment]
hello 
i have problem with my sdk channel.

i generate packages i view it but there is nothing content there.

please what happened

**Comment by Dalidadabro** [comment]
hello 
can someone help me

**Comment by ferdiworks** [comment]
If your not seeing content its because of the feed you are using most likely, Share feed link and we cxan look at it

**Comment by Dalidadabro** [comment]
Thanks for your response.

I hired someone on Fiverr to assist me in developing a channel, and they charged me $250. However, when I received the project, I could only see the logo and nothing else; my video didn't appear. After conducting some research on Google, I believe this group can help me instead.
![Attachment_1691436493](https://github.com/rrirower/DP2SceneGraphWizard/assets/141563184/23538ad1-5ad6-42f3-ba54-386cdfe3e6cc)

**Comment by Dalidadabro** [comment]
[Snack TV.zip](https://github.com/rrirower/DP2SceneGraphWizard/files/12285720/Snack.TV.zip)

**Comment by Dalidadabro** [comment]
that is the package he sent me

**Comment by Dalidadabro** [comment]
please can you help

**Comment by ferdiworks** [comment]
Thats funny I guy tried to hire me to create this channel with the Name of Frank Ojin or by name of Lin  email: Frank ojin <[linmeehoo86@gmail.com](mailto:linmeehoo86@gmail.com)> I however got the feed to work with the channel. But never delivered to him as he failed to pay me to redo his feed!

**Comment by Dalidadabro** [comment]
what? he hair you

**Comment by Dalidadabro** [comment]
i have report him on fiverr for not deliver as promised

**Comment by Dalidadabro** [comment]
please can you help to let the package show  on my TV

**Comment by rrirower** [comment]
Please do not hijack a previous issue. If your problem is unrelated, open a new issue.

**Comment by Dalidadabro** [comment]
Alright

**Comment by ferdiworks** [comment]
This is the feed for shortform video that the software did not parse https://api.npoint.io/3f4834c8754cdc939955

**Comment by Dalidadabro** [comment]
yes

**Comment by Dalidadabro** [comment]
what is the solution now

**Comment by ferdiworks** [comment]
I don't know why software doesnt parse your feed! However I did get your feed to work afteredits and work on code and created a working channel, you can contact me on my site at https://ondemandtvnetworks.com/contact.html

**Comment by ferdiworks** [comment]
Dalidababro The wizard software would not parse your shortform video feed, I create the issue here! Howsever I change code in brightscript code and got a package created and your feed parsed! As I said before contact me at my site at https://ondemandtvnetworks.com/contact.html as this is not the place for this discussion!

**Comment by Dalidadabro** [comment]
ok
i have contact you join the group i find but no response

**Comment by rrirower** [comment]
@ferdiworks the feed file you posted does not contain Category objects.  My announcement post on the Roku forum indicated that categories are required in a feed file in order for the wizard code to work.

**Comment by ferdiworks** [comment]
@rrirower I understand when we started this issue and just redid the code to work for me, Most Direct publisher channels went by the tags!

**Comment by ferdiworks** [comment]
@rrirower can you post a sample of a shortform video json feed so we are on the same sheet of music and understqanding the right or proper categorie tags

**Comment by Dalidadabro** [comment]
@rrirower please what software would we parse my short form video feed,

how can i get categories for my feed

**Comment by caloriesm** [comment]
hello

**Comment by caloriesm** [comment]
@rrirower  yes that is the exactly what happened to my channel, my channel display no content.

but After see am so happy

but After download [Channel Feed Manager software](https://github.com/rrirower/Channel-Feed-Manager) on this site it showing some error please what happened to software?
![gggftft](https://github.com/rrirower/DP2SceneGraphWizard/assets/141550793/322de3b4-8ee4-45ea-88b8-6eec102e478a)

**Comment by rrirower** [comment]
The software could not find the file "C:\\Users\\MM\\Documents\\Channel Feed Manager\\Temp\\UpdateManifest.json".  This may have been related to a problem when you installed the software.  I recommend that you uninstall and reinstall the software and try again.  If that does not fix the problem, create a dummy file called "UpdateManifest.json" within "C:\\Users\\MM\\Documents\\Channel Feed Manager\\Temp\\".

**Comment by urabore** [comment]
hello 
Am still getting the same 
 i uninstall and reinstall the software and try again and also create a dummy file but it still the something
![xdxry](https://github.com/rrirower/DP2SceneGraphWizard/assets/141771341/1e3b7392-1d0b-4d5f-b810-2d18afde9db5)

**Comment by rrirower** [comment]
Please do not update this issue with Channel Feed Manager problems.  Please continue this discussion in the [Channel Feed Manager issues #76](https://github.com/rrirower/Channel-Feed-Manager/issues/76).

**Comment by ferdiworks** [comment]
@rrirower  I know about the category tags and how they work. I been working with feeds and roku channels for a long time now. With your software first let me say thank you! I like it for simple conversions where I can help some for free and share the wealth lol. There are other channels more complicated or that have complication with converting and becomes more work as in recreating a feed file or recoding the sdk files! So as for the software I can say it has parst evrything from Series and episodes, movies ans tv specials, however I have not had a successful conversion of short form videos that parse categories. I read your replies and have not wanted to take up your time with issues. I did ask if you had a sample working feed that has category tags to view against what I have. I have talked with many other friends and developers who share code samples and sniplets that has helped with parsing a single line of videos as I already can get without their help. I can go read that Roku help files and same page you sent over and over and I can try adding category tags as seen in other samples and feeds but still to date cannot get a shortform video feed to parse categories unless I change the formats to series episodes or tvspecials and movies but not short form video! Next is there somewhere to send a pvt message if need be?

**Comment by rrirower** [comment]
@ferdiworks As I stated above, there's nothing wrong with the actual short-form videos you posted in the feed file.  The feed file is missing Category objects.  Tags are NOT Categories.  They are a way to relate a piece of content (short-form video) to a specific Category.  I've given you an example of what a Category object looks like, and I've provided an additional tool for you to use to update and manage your feed files.

I'm not sure what else I can do.
